crates/core/src/client/mod.rs

```diff
@@ -414,8 +414,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
         address: Address,
         contract_addresses: Vec<Address>,
     ) -> Result<TokenBalances, EthApiError<P::Error>> {
-        let entrypoint: Felt252Wrapper = keccak256("balanceOf(address)").try_into()?;
-        let entrypoint: FieldElement = entrypoint.into();
+        let entrypoint = FieldElement::from_byte_slice_be(&keccak256("balanceOf(address)").0[0..4])?;
```

Comment:
boyscout commit to take only the 4 first bytes of a selector in EVM world

Reply:
this seems ok for now, but I think that we should use ethers-rs to do evm related manipulation. There is probably a `get_selector` or something similar

Reply:
alright! will create an issue